structure/linkedlist/doubly.go

```diff
@@ -64,14 +64,14 @@ func (ll *Doubly) DelAtBeg() any {
 	if ll.Head != nil {
 		ll.Head.Prev = nil
 	}
-	return cur.Val
+	return cur.Val, true
 }
 
 // DetAtEnd Delete a node at the end of the linkedlist
-func (ll *Doubly) DelAtEnd() any {
+func (ll *Doubly[T]) DelAtEnd() (T, bool) {
```

Comment:
I think it makes sense to have similar signature for `Del*` and `Add*` methods. And personally, I think its better to have `error` instead of `bool`ean value.

Reply:
What do you mean, having some return on the `Add*` methods or? Don't think I agree in that case, since adding doesn't seem to have any possible faults, and returning the new added value is redundant (since it's passed in as args). Can't really agree on returning an error for the deleters either, since there appears to be only one fault scenario for those (and the fault is pretty clear)

Reply:
Sure, keep the `bool` in the return type then 😄

By having **similar** signature, I meant something like this

```Go
func (ll *Doubly[T]) DelAtEnd() (T, bool) { ... }

func (ll *Doubly[T]) AddAtEnd() bool { ... }
```
Though after your comment, I also think that it doesn't make sense to have a return type on `Add*`...

What do you guys think @raklaptudirm @siriak ?

Reply:
Oh I see 👽But yeah in that case i'll amend the things we've discussed either today or tomorrow, then make a beep

Reply:
This isn't resolved - your comment suggested that you'd change it but you haven't but marked it as resolved.

Reply:
That's what I'm confused about so I asked other maintainers @siriak @raklaptudirm but it seems they haven't seen it since you marked it as resolved.

Reply:
Aah alright, then i’ll leave it open until they’ve had the chance to give their opinions 👽

Reply:
It seems that the only possible return value is `true` for `Add*` function because there can be no errors, so it doesn't make sense to have a boolean return value as was already stated by @crunchypi